lib/app/routes/problems.rb

```diff
@@ -27,6 +28,47 @@ class Problems < Core
         problem.validate or halt 404, { error: problem.error }.to_json
         pg :problem_test, locals: { problem: problem }
       end
+
+      get '/problems' do
+        all_languages = Xapi::Config.languages # Xapi::Config.all
+        
+
+        slugs={}
```

Comment:
I'd like to have exactly one space on either side of the equals sign to make it consistent with the rest of these.